scrum-gang/neutralinojs
# Add CMake to automate build process

## Description

Added the CMake `CMakeLists.txt` file to the root of the project, which can be executed by running `cmake .` in the root directory. Also added other required code so that the build works on all 3 main operating systems (OS X, Windows and Lunix).

## Changes proposed

 - Add CMake file so that the project can be built using only one script, independent of the developer's operating system

## How to test it
1. Create a directory named `build` from the root directory
2. `cd build` then run `cmake ..`
3. Check that build succeeded and files were generated in the `build` directory

## Next steps

None.

## Deploy notes

None.
